nexus/test-utils/src/lib.rs

```diff
@@ -303,17 +281,22 @@ pub async fn test_setup_with_config<N: NexusServer>(
         user_password_hash,
     };
 
+    let tls_certificates = match initial_cert {
+        Some(cert) => vec![cert],
+        None => vec![],
+    };
```

Comment:
```suggestion
    let tls_certificates = initial_cert.into_iter().collect();
```

Reply:
Fixed in 557d218b640db9f844f930771577da7c9bb0e21b.